client/src/uri_util.ts

```diff
@@ -1,12 +1,10 @@
 // import * as Immutable from 'immutable';
 // import * as _ from 'lodash';
 import * as URI from 'urijs';
-import { QUERY_LIMIT, QUERY_MAIN_CHARS, QUERY_OFFSET, QUERY_SORT, QUERY_STAGES, QUERY_TAGS, QUERY_VS_CHARS, QUERY_PAGE_SIZE } from "./shared/query_params";
-import { getCharFilterQuery } from './shared/query_util';
+import { QUERY_LIMIT, QUERY_MAIN_CHARS, QUERY_OFFSET, QUERY_PAGE_SIZE, QUERY_SORT, QUERY_STAGES, QUERY_TAGS, QUERY_VS_CHARS } from "./shared/query_params";
```

Comment:
i see you trying to slip in your java double quotes in here

Reply:
You got me